python/tests/test_tables.py

```diff
@@ -2368,12 +2368,13 @@ def wf_sim_with_individual_metadata(self):
             9,
             10,
             seed=1,
-            deep_history=True,
+            deep_history=False,
```

Comment:
I haven't thought throught he implications of changing this for `test_shuffled_individual_parent_mapping`, have you?

Reply:
Erm, why should this make a difference. We still shuffle the individuals and check that they are (a) shuffled and (b) the original individual ids correspond.

Reply:
But perhaps I'm not understanding something?

Reply:
Why change the default here? Seems gratuitous to me.

Reply:
See below. If we want to check that all nodes have an individual, and all the individuals point to their correct parents, then we can't include the deep history nodes, which have no individuals (and the nodes at the top of the WF generated simulation will not have the correct parents.

I.e. if we have a deep history, we (almost by definition) cannot fully map the trees onto the individuals pedigree. It seems worth testing that we were capturing *all* of the genetic pedigree in the individuals.

Reply:
(yan has thought this over)